scripts.js

```diff
@@ -102,4 +85,54 @@ const resetBoard = () => {
     // sets the innerHTML to null to replace the "X" or "O"
```

Comment:
I dont see a reset for the board variable. you'll want to add it to this function:
```
board = [
  ['','',''],
  ['','',''],
  ['','',''],
]
```